content/es/site-reliability-engineering.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Site Reliability Engineering
+status: Completed
+category: Concepto
+tags: ["methodology", "", ""]
+---
+
+## ¿Qué es?
+
+Site Reliability Engineering o SRE es una disciplina que combina operaciones e ingeniería de software.
+Este último se aplica a problemas de infraestructura y operaciones, específicamente.
+Es decir, en lugar de crear características del producto, los ingenieros de confiabilidad del sitio crean sistemas para ejecutar aplicaciones.
+Hay similitudes con [DevOps](/es/devops/), pero mientras DevOps se enfoca en llevar el código a producción,
+SRE garantiza que el código que se ejecuta en producción funcione correctamente.
+
+## Problema que aborda
+
+Asegurar que las aplicaciones se ejecuten [confiablemente](/es/reliability/) requiere múltiples capacidades,
+desde la supervisión del rendimiento, las alertas, la [depuración](/es/debugging/) hasta la resolución de problemas.
+Sin estos, los operadores del sistema solo pueden reaccionar a los problemas en lugar de trabajar de manera proactiva para evitarlos.
```

Comment:
Remove the final dot. This dot is not in the original English file and this was added automatically by using a machine translator:

```suggestion
Sin estos, los operadores del sistema solo pueden reaccionar a los problemas en lugar de trabajar de manera proactiva para evitarlos
```